src/test/java/nc/students/ayaz/model/UserTest.java

```diff
@@ -19,15 +20,30 @@ public void shouldOwnVideoWhenVideoAdded() {
     }
 
     @Test
-    public void shouldGetVideoWhenPassedVideoName() throws Exception {
+    public void shouldGetVideoByNameWhenItWasAddedEarlier() throws Exception {
```

Comment:
&#x1F34E; **Issue**
shouldGetVideoByNameWhenItWasAddedEarlier -> shouldBeAbleToGetVideoByNameWhenItWasAddedEarlier
Otherwise its not precise enough.